src/main/java/org/wso2/carbon/apimgt/securityenforcer/utils/AISecurityHandlerConstants.java

```diff
@@ -47,6 +47,7 @@
     public static final int ASE_RESPONSE_CODE_INCORRECT_JSON = 400;
     public static final int ASE_RESPONSE_CODE_UNKNOWN_API = 503;
     public static final int ASE_RESPONSE_CODE_UNAUTHORIZED = 401;
+    public static final int ASE_RESPONSE_CODE_FORBIDDEN = 403;
```

Comment:
Can't we use `org.apache.commons.httpclient.HttpStatus` for these constants?

Reply:
These are specific responses from Ping ASE. It may change from generic HTTP status codes.